docs/guide/testrunner/debugging.md

```diff
@@ -71,6 +71,33 @@ node-inspector --debug-port 5859 --no-preload
 
 The `no-preload` option defers loading source file until needed, this helps performance significantly when project contains a large number of node_modules, but you may need to remove this if you need to navigate your source and add additional breakpoints after attaching the debugger.
 
+## Node built-in debugging with chrome-devtools
+
+Chrome devtool debugging looks like it's going to be the accepted replacement for node-inspector. This quote is from the node-inspector github README:
+
+> Since version 6.3, Node.js provides a buit-in DevTools-based debugger which mostly deprecates Node Inspector, see e.g. this blog post to get started. The built-in debugger is developed directly by the V8/Chromium team and provides certain advanced features (e.g. long/async stack traces) that are too difficult to implement in Node Inspector.
```

Comment:
> Node.js provides a buit-in DevTools-based debugger which mostly deprecates Node Inspector

Can we add a note to the "Node Inspector" section saying that if you use Node v6.3 or up use "Node built-in debugging with chrome-devtools" instead because NodeInspector will be deprecated .. or something along these lines